src/pybind/mgr/dashboard/frontend/src/app/shared/models/cd-notification.ts

```diff
@@ -1,5 +1,20 @@
+import { ToastOptions } from 'ng2-toastr';
 import { NotificationType } from '../enum/notification-type.enum';
 
+export class CdNotificationConfig {
```

Comment:
Would it possibly make sense to move all the changes related to ``CdNotificationConfig`` to a separate (pre-)PR? Because this one is quite long already.

Reply:
Done :)